DelvUI/Interface/Jobs/DarkKnightHud.cs

```diff
@@ -43,6 +43,12 @@ protected override (List<Vector2>, List<Vector2>) ChildrenPositionsAndSizes()
                 sizes.Add(Config.BloodGaugeSize);
             }
 
+            if (Config.ShowDarkside)
+            {
+                positions.Add(Config.Position + Config.DarksidePosition);
+                sizes.Add(Config.DarksideSize);
+            }
+
             if (Config.ShowBuffBar)
```

Comment:
Probably shouldnt add this bar for draggable area calculations since its not always showing.

Reply:
I planned on making it optional wether the bar always shows or "only show when active" much like Zeff did with bard dots. My plan was actually to get around to making that an option for most elements in the jobpacks that are triggered and have a timer